test/test.js

```diff
@@ -603,6 +603,12 @@ suite('Response', function() {
 
     assert.equal(r.headers.get('content-type'), 'text/plain')
   })
+
+  test('accepts options as first parameter', function () {
+    var r = new Response({ status: 200 })
```

Comment:
Let's use a non-default status such as 204 to verify that the status was successfully assigned.